howto/manage/setup-tls-for-appliance.md

```diff
@@ -0,0 +1,81 @@
+
+[note type="information" status="Note"]This tutorial assumes you have Anbox Cloud Appliance installed. If you haven't, please follow the [instruction](https://discourse.ubuntu.com/t/install-the-anbox-cloud-appliance/22681) to do so.[/note]
+
+## Add a DNS Record
+
+Setting up DNS redirection depends on your DNS provider. Refer to the documentation of your provider to create a DNS record pointing to the IP/DNS of the AWS instance where the Anbox Cloud appliance is running on.
+
+## Configure the location
+
+Configure the location for the appliance using the created DNS name.
+
+    sudo snap set anbox-cloud-appliance experimental.location=<your DNS name>
+
+## Generate an SSL certificate
+
+There are many ways to create a valid SSL certificate. It's recommended to use [Let's Encrypt](https://letsencrypt.org/) to generate a free SSL certificate.
+
+First, connect and SSH into your appliance instance, and install the `certbot` snap:
+
+    sudo snap install --classic certbot
+
+Before generating your certificate using `certbot`, you need to stop the traefik service from listening on port 80 for the certificate creation.
+
+    sudo snap stop anbox-cloud-appliance.traefik
+
+Then run the following command to generate your certificate:
+
+    sudo certbot certonly --standalone
+
+It prompts you to enter the domain name for the certificate to be generated. You will see the following message when the certificate is created successfully.
+
+```bash
+Successfully received certificate.
+Certificate is saved at: /etc/letsencrypt/live/<your domain name>/fullchain.pem
+Key is saved at:         /etc/letsencrypt/live/<your domain name>/privkey.pem
+This certificate expires on yyyy-MM-dd.
+These files will be updated when the certificate renews.
+Certbot has set up a scheduled task to automatically renew this certificate in the background.
+```
+
+## Install an SSL certificate
+
+Copy the generated certificate to the `/var/snap/anbox-cloud-appliance/common/traefik/tls` directory
+
+    sudo cp /etc/letsencrypt/live/<your domain name>/fullchain.pem /var/snap/anbox-cloud-appliance/common/traefik/tls/cert.pem
+    sudo cp /etc/letsencrypt/live/<your domain name>/privkey.pem   /var/snap/anbox-cloud-appliance/common/traefik/tls/key.pem
+
+Start the traefik service afterwards
+
+    sudo snap start anbox-cloud-appliance.traefik
+
+With the certificate installed on the appliance, you now can access it using the created domain name.
+
+## Renew an SSL certificate
+
+The `certbot` snap packages installed on your machine already setup a systemd timer that will renew your certificate automatically before they expire. However in order to get the certificate renewed successfully for the appliance, you need to
```

Comment:
We should warn the user that the automatic renewal will mean a short (random) downtime of his appliance (e.g. traefik wont be accessible)